src/Sinch/Conversation/Webhooks/WebhookTrigger.cs

```diff
@@ -0,0 +1,36 @@
+/// </summary>
+[JsonConverter(typeof(EnumRecordJsonConverter<WebhookTrigger>))]
+public record WebhookTrigger(string Value) : EnumRecord(Value)
+{
+   
+    public static readonly WebhookTrigger UnspecifiedTrigger = new("UNSPECIFIED_TRIGGER");
+    public static readonly WebhookTrigger MessageDelivery = new("MESSAGE_DELIVERY");
+    public static readonly WebhookTrigger MessageSubmit = new("MESSAGE_SUBMIT");
+    public static readonly WebhookTrigger EventDelivery = new("EVENT_DELIVERY");
+    public static readonly WebhookTrigger MessageInbound = new("MESSAGE_INBOUND");
+    public static readonly WebhookTrigger SmartConversation = new("SMART_CONVERSATION");
+    public static readonly WebhookTrigger MessageInboundSmartConversationRedaction = new("MESSAGE_INBOUND_SMART_CONVERSATION_REDACTION");
+    public static readonly WebhookTrigger EventInbound = new("EVENT_INBOUND");
+    public static readonly WebhookTrigger ConversationStart = new("CONVERSATION_START");
+    public static readonly WebhookTrigger ConversationStop = new("CONVERSATION_STOP");
+    public static readonly WebhookTrigger ContactCreate = new("CONTACT_CREATE");
+    public static readonly WebhookTrigger ContactDelete = new("CONTACT_DELETE");
+    public static readonly WebhookTrigger ContactMerge = new("CONTACT_MERGE");
+    public static readonly WebhookTrigger ContactUpdate = new("CONTACT_UPDATE");
+    public static readonly WebhookTrigger Unsupported = new("UNSUPPORTED");
+    public static readonly WebhookTrigger OptIn = new("OPT_IN");
+    public static readonly WebhookTrigger OptOut = new("OPT_OUT");
+    public static readonly WebhookTrigger Capability = new("CAPABILITY");
+    public static readonly WebhookTrigger ChannelEvent = new("CHANNEL_EVENT");
+    public static readonly WebhookTrigger ConversationDelete = new("CONVERSATION_DELETE");
+    public static readonly WebhookTrigger ContactIdentitiesDuplication = new("CONTACT_IDENTITIES_DUPLICATION");
+    public static readonly WebhookTrigger SmartConversations = new("SMART_CONVERSATIONS");
```

Comment:
I'm not sure both `SMART_CONVERSATION` and `SMART_CONVERSATIONS` exist. Can you double-check?

Reply:
That's what I get from oas spec, 
`SMART_CONVERSATION` exists but it's in beta 
![image](https://github.com/sinch/sinch-sdk-dotnet/assets/12355173/7a5e5397-c9e9-44af-896f-474c477116d3)

Reply:
I'll remove plural variant